internal/client.go

```diff
@@ -177,6 +213,117 @@ func (c *client) reportStatus(ownerLogin, repoName, statusesURL, status, descrip
 	return nil
 }
 
+func (c *client) reportDismissedReviews(owner, repo string, prNumber int, reviewers []string) error {
+	if len(reviewers) == 0 {
```

Comment:
Maybe it's my personal preference, but I find it a bit unusual that the client fails silently when `reviewers` is an empty slice.